src/plugins/controls/public/services/data/types.ts

```diff
@@ -6,24 +6,17 @@
  * Side Public License, v 1.
  */
 
-import { Observable } from 'rxjs';
+// import { Observable } from 'rxjs';
```

Comment:
It looks like this line can be removed?

Reply:
Yesss, you're totally right - thank you for catching that 👍